### ISSUE TITLE
No Zoom Buttons on Maps in Embedded Groups

### ISSUE THREAD

**Issue by sineadfenton** [openfoodfoundation/openfoodnetwork#2723]
## Description

Users are finding it difficult to interact with the embedded groups map. Users and UK team have reported that the map should have +/- icons to zoom in and out of the map and not as current functionality dictates where  (cntrl+wheel have to be used)

## Expected Behavior

icons in the right bottom to zoom in and out with

## Actual Behavior

no icons to zoom therefore making the user experience difficult. The mouse and keyboard have to be used

## Steps to Reproduce

1. Go to an embedded map - https://www.biodynamic.org.uk/where-to-buy/
2. See lack of icons in bottom right
3. Use mouse and keyboard to scroll - unfriendly process
4.

## Animated Gif/Screenshot

![image](https://user-images.githubusercontent.com/28807449/45517961-cce85600-b7a7-11e8-8896-da96c1ceb831.png)

## Context

Users have a bad experience when using the map

## Your Environment

* Version used: 
* Browser name and version: Chrome 68
* Operating System and version (desktop or mobile): Desktop 

## Possible Fix

Add the zoom icons

**Comment by sineadfenton** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Can't link to producers on BDA embedded map and map is missing zoom buttons](https://trello.com/c/glOHszYd/190-cant-link-to-producers-on-bda-embedded-map-and-map-is-missing-zoom-buttons)

**Comment by daniellemoorhead** [comment]
This seems to be UK-specific - I looked on .org.au and we have zoom controls (we also have a strange error dialogue box showing up, but that's a different problem).

@RachL @sigmundpetersen @sauloperez can you check if you get the zoom controls or not?

**Comment by daniellemoorhead** [comment]
Just one other update: I get the zoom controls when I go onto .org.uk on Chrome/Mac desktop

![screen shot 2018-09-14 at 4 34 56 pm](https://user-images.githubusercontent.com/10949052/45533832-53e70e00-b83c-11e8-8004-b020fc4cac08.png)

**Comment by lin-d-hop** [comment]
@daniellemoorhead note that this issue affects embedded groups. This is not an issue referring to the UK map. I'll make that more clear in the description :-)

**Comment by daniellemoorhead** [comment]
Well then, that makes much more sense!

**Comment by sauloperez** [comment]
I checked https://www.biodynamic.org.uk/where-to-buy/ and there's something wrong with the CSS/HTML of the iframe. No matter what height we set to the iframe, the bottom part of the map where controls are rendered is always cut.

**Comment by RachL** [comment]
@sauloperez yes long time issue. But I don't think we are maintaining our iframes anymore...

**Comment by sauloperez** [comment]
@lin-d-hop can biodynamic live with this? If so, we can close. It was a perfect issue for a new FE dev, that's why.

**Comment by lin-d-hop** [comment]
They would still like this feature, despite the fact that they've been living without it for like 2 years :-)

Sounds like should use it for FE entry.

**Comment by RachL** [comment]
Wouldn't it be better to find a solution to replace the current iframes? @lin-d-hop @sauloperez I mean, look at this page on mobile 🙈

**Comment by lin-d-hop** [comment]
Obviously it would be better to find a solution to replace iframes. If we see it in less than 2 years our productivity will have increased by like 1000%

In the mean time I believe Pau is short of intervew GH issues for the FE dev, is that correct?

**Comment by RachL** [comment]
As you wish, I'm just a bit afraid also this is not highly motivating for a senior FE dev... but maybe I'm wrong

**Comment by andresgutgon** [comment]
I think is an issue with this page Javascript
![image](https://user-images.githubusercontent.com/49499/92326027-67a26500-f04f-11ea-8163-2265c362e28f.png)

☝️ My theory is that they are not loading ok jQuery

If you go to this same map out of this page the controls are there:
https://openfoodnetwork.org.uk/groups/biodynamic-association-licensees?embedded_shopfront=true#/map

**Comment by andresgutgon** [comment]
☝️  I made a PR for it

Ok, after more investigation I see this error is provoked [by our code here](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/views/layouts/_shopfront_script.html.haml#L2)

Maybe this error on our side is not the culprit for missing +/- controls on the map. But I think fixing it doesn't hurt

**Comment by luisramos0** [comment]
I am re-opening this for the prod test after 5996 is deployed.

**Comment by RachL** [comment]
Hello @andresgutgon are you still working on this one? :-)

**Comment by andresgutgon** [comment]
So sorry, un assign me please
